src/util/Timer.cpp

```diff
@@ -153,6 +153,12 @@ VirtualClock::tmToSystemPoint(tm t)
     return VirtualClock::system_time_point() + std::chrono::seconds(tt);
 }
 
+std::time_t
+VirtualClock::tm_to_time_t(std::tm tm)
```

Comment:
Two issues here:

  - naming style: `tmToTimeT`
  - include mention of the fact that this is going through system-time rather than steady-time: `tmToSystemTimeT`

Reply:
Name changed.